src/main/java/com/quorum/gauge/common/QuorumNetworkProperty.java

```diff
@@ -296,12 +295,24 @@ public void setName(String name) {
         public void setAccountAliases(Map<String, String> accountAliases) {
             this.accountAliases = accountAliases;
         }
+
+        public String getPsi() {
+            return psi;
+        }
+
+        public void setPsi(String psi) {
+            this.psi = psi;
+        }
     }
 
     public static class DockerInfrastructureProperty {
         private boolean enabled;
         private String host;
         private Map<String, DockerContainerProperty> nodes = new HashMap<>();
+        // Quorum image provided by Terraform var.quorum_docker_image
+        private String targetQuorumImage;
```

Comment:
this is a way to modify the `develop` image by like a test image?

Reply:
As far as I know it is reading the contents of the application-docker.yml generated by terraform:
```yaml
quorum:
  consensus: raft
  docker-infrastructure:
    enabled: true
    target-quorum-image: quorumengineering/quorum:develop
    target-tessera-image: quorumengineering/tessera:develop
    nodes:
      Node1:
        quorum-container-id: 14b154b5270d56653f4d5865cda816b75b8d3a1cf1b12ea82f2881bab6ce7f50
        tessera-container-id: 13a13860254e3f29532bcbfce515f034b2440a9e606b62f839fcdff26af08d44
```
When template networks are used it is possible to stop the container and update the image being used while preserving the storage (see the upgrade tests).